### ISSUE TITLE
Add option for snaplen

### ISSUE THREAD

**Issue by tlnd** [travelping/docker-pcap#15]
It should be possible to truncate captured packets by supporting the `-s <snaplen>` option.

**Comment by KilianDargel** [comment]
When looking for a suitable default for `-s <snaplen>` I found in the wireshark [documentation](https://wiki.wireshark.org/SnapLen)

>For example, the standard tool TcpDump will by default use a SnapLen of 68 bytes for IPv4 packets and 96 bytes for IPv6 packets. 

I'll test this.

**Comment by KilianDargel** [comment]
Snaplength will have *no* default in order not to break anything. It's checked conditionally if it's used. If blank, it won't use the `-s` parameter otherwise.
```if [ -n "$SNAPLENGTH" ];
then
  SNAPLENGTH="-s $SNAPLENGTH"
fi

/usr/bin/tshark ... $SNAPLENGTH```

**Comment by tlnd** [comment]
Closed with 657e961129e32fae61fe8d36bbe5d60f7a59462d
